### ISSUE TITLE
Outreachy Documentation Project: <ElizabethJoseph>

### ISSUE THREAD

**Issue by ElizabethMawutin** [ersilia-os/ersilia#114]
Applicant: <applicant github handle>
  
Welcome to the Ersilia Open Source Initiative. This issue will serve to track all your contributions for the project “Improve the documentation and outreach material of the Ersilia Model Hub”.
  
Please tick the tasks as you complete them. To make a final application it is *not* required to have completed all tasks. Only the *Initial Steps* and *Community* sections are REQUIRED. The tasks are not ordered from more to less important, they are simply related to different skills. Start where you feel most comfortable. This project can be adapted to the applicants interests, please focus on the type of tasks that you prefer / have better skills / would like to work on as an intern.

---
### Initial steps:
- [x] Record your application for the project in the outreachy website referencing this issue. Please make sure to select the right project on the website.
- [x] Join the Slack channel to follow public communications
- [x] Comment under this issue explaining why are you interested in this project
---
### GitHub documentation:
- [x] Create a README file with the name <README_yourname.md> under the /documentation folder
- [x] Link the #PR in a comment under this issue
- [x] Incorporate feedback from the mentor
---
### Writing dissemination material
- [x] Read the [Strategic plan 2021-2023](https://ersilia.gitbook.io/ersilia-wiki/) for Ersilia and create a 1-page blogpost with the main points
- [x] Comment under this issue with a link to the blogpost (a google docs for example)
- [x] Incorporate feedback from the mentor
- [x] Choose your own topic related to Ersilia (AI/ML for biomedical research, neglected diseases, drug discovery…) and write a 1-page blogpost to communicate to a non-expert audience
- [x] Comment under this issue with a link to the blogpost (a google docs for example)
- [x] Incorporate feedback from the mentor
- [x] Create a template for a twitter post to release every time a new model is incorporated in the Hub (twitter: 280 characters, you can suggest a main post +  thread with extra information) and add it as a comment under this issue
- [x] Create a template short Newsletter (1 paragraph) to send every month to our community (funders, users, contributors). It should mention metrics (models in the hub, number of users, funding…), thank you etc 
---
### Technical skills (required for the tutorial only)
- [x] [Install](https://ersilia.gitbook.io/ersilia-book/quick-start/installation) the Ersilia Model Hub
- [x] Test one model
- [x] Add a screenshot under this issue showing the model running in your computer
- [ ] Write a docstring for the ErsiliaModel class. Use the Google Python Style guide. Paste the docstring as a comment below (do not use a PR).
---
### Graphic material
- [x] Read the [Ersilia Brand Guidelines](https://medium.com/ersiliaio/ersilia-open-source-initiative-brand-guidelines-130fe53104f5)
- [x] Read [“Why Ersilia?”](https://ersilia.gitbook.io/ersilia-wiki/)
- [x] Create one image / slide to explain Ersilia’s mission and vision
- [x] Link to the image/slide as a comment under this issue
- [x] Incorporate feedback from the mentor
- [ ] Create two slides / short video showing how to use the Ersilia Model Hub and add them under the /tutorial folder 
- [ ] Link the #PR in this issue
- [ ] Incorporate feedback from the mentor
---
### Scientific content
- [x] Check the models available in the [Hub](https://airtable.com/shrXfZ8pqro0jjcsG/tblZGe2a2XeBxrEHP/viwd5XJVLslkE11Tg)
- [ ] Select one model from the list and write a technical card (what is the model for, what input, which data was used to create it, what kind of ML algorithm uses…) for it
- [ ] Add your card as a comment to this issue
- [ ] Search the scientific literature and suggest 3 new models (comment in this issue) that would be relevant to incorporate in the Hub.
---
### Community
- [x] Look up two other projects and comment on their issues with feedback on one of their tasks
- [x] If you have feedback from your peers, answer it in this issue.
---
### Other
If you have interest in working on related topics, or have new suggestions, please do the following
- [ ] Add a comment in this issue with your new idea, tagging the mentor 
- [ ] Get feedback from the mentor and act accordingly
- [ ] Link in the comments any other PR you have contributed to.
---
### Final application
- [x] I have answered all comments from mentors and contributors
- [x] All PR or issues assigned to me are complete
- [x] I have submitted my final application to the project

**Comment by ElizabethMawutin** [comment]
Hello @GemmaTuron .
Here is the [Readme](https://github.com/ersilia-os/ersilia/commit/2dac58b6a35bae0bcd5250c832cce6282d5a81f7) file i created.
Below are the contributions i made on the file.
* I added the project aim to the project description
* I added a screenshot of the schematic view of Ersilia Model Hub
* I listed the pre-requisites for installing the model
* I added a link to the GitHub profile of the contributors
* I added the programming language used by the model
* I added links to joining our workspace and subscribing to our mailing list.
Please do review and revert if there is need for any change. Thank you.

**Comment by GemmaTuron** [comment]
Hi @ElizabethMawutin,

Thanks for the work, I have gone through your document but before giving more comments can you do the following change:
- When you create a file you must specify the type of document, otherwise it cannot be opened as markdown. In this case, it should be: README_ElisabethJoseph.md
Then I will be able to visualize with the full markdown formatting which will be much better!

**Comment by ElizabethMawutin** [comment]
Hello @GemmaTuron, here is the corrected [README](https://github.com/ersilia-os/ersilia/pull/136/commits/9139cf732bf20ef24a903320e73c2d8c970fb359) file. 
While awaiting your response on this issue, can i treat other issues on the template?.

**Comment by GemmaTuron** [comment]
Hi @ElizabethMawutin 

Thanks, I only have a few comments! I like the image but I think its too complex for a readme file... its just my opinion.
Double check which license we are using, and in the follow us section do not add the slack link as it is only for outreachy, but perhaps add twitter and linkedin accounts?

**Comment by ElizabethMawutin** [comment]
Hello @GemmaTuron ,
Please find the attached files for my blog post.
1. My blog post for [Ersilia Strategic Plan 2021 - 2023](https://docs.google.com/document/d/1zfbquAkS045sXuoaTtllGv_amf22RUDrCUu3o3bTyQo/edit)
2. My blog post on [Neglected diseases](https://docs.google.com/document/d/1xnY9lFKVq-mTTPH0j8v23ig5XH09pZ7AupCk1CM7aPE/edit?usp=sharing)

Thank you.

**Comment by ElizabethMawutin** [comment]
Hello @GemmaTuron . Here is a screenshot of the Model running on my pc.

![ubu 2](https://user-images.githubusercontent.com/102416858/162588909-578f05a5-d2dc-4288-9b8f-1e25b6215eb6.PNG)
![ubu 3](https://user-images.githubusercontent.com/102416858/162588907-012c16f3-87dc-44c7-8595-612dc4bc8628.PNG)
Thank you.

**Comment by ElizabethMawutin** [comment]
Hello @GemmaTuron, Here is my Twitter post template.

**New Model Alert!**
Hello **scientists,** we’ve just added a new **Model** to our Hub. 
I present to you **(Model name).** Want to know what this model does?, follow the thread……
This model is designed to (brief description of the model). And that’s not all ….
This model can be used to (explain functions of the model)
Want to know more about this model?, visit (website)

**Comment by ElizabethMawutin** [comment]
Hello @GemmaTuron,
Here is a link to my [Newsletter](https://www.canva.com/design/DAE9kYeZr1Q/v1OdD1N72wO6NW2xAY2G5Q/edit?layoutQuery=Newsletter#) template for our Users, Contributors and Funders.
Please do review and revert if there is need for changes.
Thank you.

**Comment by GemmaTuron** [comment]
Hi @ElizabethMawutin 

Thanks for all the work. I am adding here minor comments:
- Strategic plan: sentences are very long, which makes it difficult sometimes to follow. 
- Blogpost: nice summary of neglected diseases, I would rephrase the developing nations (Africa, Asia .. continents? or refer to Low Income Countries in general)
- Canva: if you can reduce text in the vision mission that would be fantastic

Overall I think you have done a great job, and are ready for final application. Check the comments above, comment here is there is any other task you are working on and remember to link this issue in your final application!

**Comment by ElizabethMawutin** [comment]
Hello @GemmaTuron. Thank you for your comments, i'll make the necessary changes as soon as possible and revert back to you.
Am presently working on creating a tutorial on how to use Ersilia Model and writing a technical card for the model.

**Comment by ElizabethMawutin** [comment]
Hello @GemmaTuron. I have incorporated your feedback on the tasks. Below are the links to the corrected tasks.
1. [Ersilia Strategic Plan](https://docs.google.com/document/d/1zfbquAkS045sXuoaTtllGv_amf22RUDrCUu3o3bTyQo/edit)
2. [Neglected Diseases](https://docs.google.com/document/d/1xnY9lFKVq-mTTPH0j8v23ig5XH09pZ7AupCk1CM7aPE/edit)
3. [Ersilia Mission and Vision](https://www.canva.com/design/DAE9lMQZ5Hc/CNB781xkYS4fVxguBDFB4Q/edit)
Thank you.

**Comment by victorabba** [comment]
Nice work you have there @ElizabethMawutin

**Comment by ElizabethMawutin** [comment]
Thank you @victorabba @ifeoluwafavour

**Comment by tracycod3r** [comment]
Great work @ElizabethMawutin

**Comment by Jaya3112** [comment]
Well done @ElizabethMawutin .Keep doing.

**Comment by GemmaTuron** [comment]
Hi @ElizabethMawutin 

Thanks for the changes! Please focus on finishing off tasks and submitting final applications

**Comment by ElizabethMawutin** [comment]
Thank you @Jaya3112 @tracycod3r

**Comment by ElizabethMawutin** [comment]
Hello,
@GemmaTuron i am having difficulty making my video tutorial so i'll be stopping here for now. Can i still send the tutorial after rectifying the issue am having?.

**Comment by GemmaTuron** [comment]
Hi @ElizabethMawutin 

Thanks for letting me know. No problem you have already done a lot of work so you have plenty of material to write your final application!

**Comment by ElizabethMawutin** [comment]
Thank you @GemmaTuron.
